packages/cosmwasm/src/multisigcosmwasmclient.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { BroadcastMode, GasLimits, GasPrice, OfflineSigner } from "@cosmjs/launchpad";
+
+import { CosmWasmFeeTable, ExecuteResult, SigningCosmWasmClient } from "./signingcosmwasmclient";
+
+export type Duration =
+  | {
+      readonly at_height: number;
+    }
+  | {
+      readonly at_time: number;
+    };
+
+export enum Vote {
+  Yes = "yes",
+  No = "no",
+  Abstain = "abstain",
+  Veto = "veto",
+}
+
+export class MultisigCosmWasmClient extends SigningCosmWasmClient {
```

Comment:
What about renaming this to `Cw3CosmWasmClient`? Looking at https://github.com/cosmos/cosmjs/issues/416#issuecomment-718904679 and #501 there seems to be some variety of multisig solution upcoming and right now I don't see how to magically abstract away all of this. But we can have a bunch of this wrapper classes, which can be used directly or serve as a template for creating a similar but incompatible one.